Assets/Scripts/WheelMotor.cs

```diff
@@ -14,17 +15,26 @@ public class WheelMotor : Motor
 
     private WheelCollider _wheel;
 
+    [SerializeField]
+    private float brakeTorque;
+
     protected override void Awake()
     {
         base.Awake();
         _wheel = GetComponent<WheelCollider>();
+        brakeTorque = Mathf.Infinity;
```

Comment:
This should be set in the editor, not hardcoded in `Awake()`

Reply:
ayush says he fixed this